backends/platform/sdl/win32/win32.cpp

```diff
@@ -65,7 +65,7 @@ void OSystem_Win32::init() {
 
 void OSystem_Win32::initBackend() {
 	// Console window is enabled by default on Windows
-	ConfMan.registerDefault("console", true);
+	ConfMan.registerDefault("console", false);
```

Comment:
This does not belong here

Reply:
@sev- : This and the other irrelevant commits have been removed in my refactoring of this PR as:
https://github.com/digitall/scummvm/tree/pr931_fixup

I have also fixed the formatting as per your next commit. I can't force push over this PR AFAIK as I am not a collaborator to @henke37 on this.

I do not know the intention and advantages of this, so would need him to respond, but if not we can close this and create a new PR from my fixup branch if needs be.